src/python/pants/backend/python/goals/pytest_runner.py

```diff
@@ -204,6 +214,31 @@ def _count_pytest_tests(contents: DigestContents) -> int:
     return sum(len(_TEST_PATTERN.findall(file.content)) for file in contents)
 
 
+async def validate_pytest_cov_included(_pytest: PyTest):
```

Comment:
Why private var?

Reply:
Because it was shadowing the imported name `pytest`, and causing confusion.